### ISSUE TITLE
Yubico Auth app for Windows cannot read QR code from epicgames.com

### ISSUE THREAD

**Issue by BitzerMaloney** [Yubico/yubioath-flutter#826]
* Yubico Authenticator version: 6.0.0
* Operating system and version: Win 10 Pro, version 21H1 (19043.2251)
* YubiKey model and version: YubiKey 5 NFC
* Bug description summary: Yubico Auth app for Windows cannot read QR code from epicgames.com

**Steps to reproduce**
Attempt to enable 2FA at epicgames.com. Use Windows version of Yubico Authenticator app do do this.

**Expected result**
QR code is scanned and Yubico Authenticator app adds new account.

**Actual results**
Authenticator app is not able to find QR code on the screen - it displays error "No QR code found". Retries does not resolve this issue. I also returned to home page of epicgames.com and tried again hoping that new QR code would work, but without success. QR code scans into other apps fine (e.g. Google Authenticator).

**Other info**
I have used the Windows version of Yubico Authenticator to scan codes from other sites, so the issue seems specific to epicgames.com.

**Comment by dainnilsson** [comment]
We've looked at the QR code generated by Epic, and the reason it isn't scanning on desktop is that it doesn't follow the specification properly, making it more difficult to scan. Specifically, the main issue is around the rounded corners on the tracking squares:

![image](https://user-images.githubusercontent.com/140781/203309159-13c7c455-72f2-4859-b12d-ea0f71616981.png)

Out of curiosity I took a screenshot and manually "filled in" the corners in Paint, and then the QR scans just fine.

Ideally this is something that Epic should address on their end.

We use a third party library (https://github.com/zxing-cpp/zxing-cpp) for QR scanning. I don't know if they'd be willing to consider this an issue worth fixing or not on their end. If they do, then that change would be reflected in our scanner as well.
